src/grunt/helper.js

```diff
@@ -111,34 +111,41 @@ ns.init = function (grunt) {
                 reporters: ['progress', 'coverage'],
 
                 coverageReporter: {
-                    reporters: [{
-                        type: 'text-summary'
-                    }, {
-                        type: 'html',
-                        dir: path.join(process.cwd(), 'coverage'),
-                    }],
+                    reporters: [
+                        {
+                            type: 'text-summary',
+                        },
+                        {
+                            type: 'html',
+                            dir: path.join(process.cwd(), 'coverage'),
+                        },
+                    ],
```

Comment:
@psbanka I was afraid you'd say that... I guess I'll see if there's an option in the new `comma-dangle` that matches the current behavior in `no-comma-dangle` (the latter has been deprecated and will be removed in v1.0). 

Then again, it is 3-1 against you...